src/templates/antora/util.ts

```diff
@@ -0,0 +1,262 @@
+  const fileName = (pageFile.path === 'tinymce' ? getRootPath(structure) : pageFile.path) + '.adoc';
+  const folder = getNamespaceFromFullName(pageFile.path);
+  return generateNavXref('api/' + folder, fileName, getNameFromFullName(pageFile.title));
+};
+
+const getRootPath = (structure: ExportStructure): string => {
+  switch (structure) {
+    case 'legacy':
+      return 'root_tinymce';
+
+    case 'default':
+      return 'tinymce.root';
+  }
+};
+
+const getNamespacesFromTypes = (types: Type[]): Record<string, string> => {
+  return types.reduce((namespaces: Record<string, string>, type: Type) => {
+    const fullName = type.fullName.toLowerCase();
+    const url = getNamespaceFromFullName(fullName);
+    if (url && !namespaces[url]) {
+      namespaces[url] = getNamespaceFromFullName(type.fullName);
+    }
+    return namespaces;
+  }, {});
+};
+
+const getDescriptionsFromTypes = (types: Type[]): Record<string, string> => {
+  return types.reduce((descriptions: Record<string, string>, type: Type) => {
+    const name = type.fullName.toLowerCase();
+    if (name && !descriptions[name]) {
+      descriptions[name] = type.desc;
+    }
+    return descriptions;
+  }, {});
+};
+
+const getNavFile = (types: Type[]): NavFile => {
+  const namespaces = getNamespacesFromTypes(types);
+  const pages = Object.entries(namespaces).map(([ url, title ]): NavFile => {
+    const innerPages = types.filter((type) => {
+      const fullName = type.fullName.toLowerCase();
+      return getNamespaceFromFullName(fullName) === url;
+    }).map((type): NavFile => {
+      return { title: type.fullName, path: type.fullName.toLowerCase() };
+    });
+
+    return {
+      title,
+      path: url,
+      pages: innerPages
+    };
+  });
+
+  return {
+    title: 'API Reference',
+    path: BASE_PATH,
+    pages
+  };
+};
+
+const navToAdoc = (indexPage: NavFile, structure: ExportStructure): string => {
+  // Api index nav page top level
+  let adoc = navLine(indexPage.title, 1);
+  if (indexPage.pages) {
+    indexPage.pages.forEach((namespace) => {
+      adoc += navLine(namespaceNavLine(namespace, structure), 2);
+      if (namespace.pages) {
+        namespace.pages.forEach((pageFile) => {
+          adoc += navLine(pageFileNavLine(pageFile, structure), 3);
+        });
+      }
+    });
+  }
+  return adoc;
+};
+
+const generateNavPages = (indexPage: NavFile, structure: ExportStructure): PageOutput[] => {
+  const navPages = [] as PageOutput[];
+  navPages.push({
+    type: 'adoc',
+    filename: '_data/nav.yml',
+    content: YAML.dump(indexPage)
+  });
+
+  const adocNav = navToAdoc(indexPage, structure);
+  navPages.push({
+    type: 'adoc',
+    filename: '_data/moxiedoc_nav.adoc',
+    content: adocNav
+  });
+  return navPages;
+};
+
+const legacyIndexToAdoc =
+(namespace: NavFile, template: HandlebarsTemplateDelegate, descriptions: Record<string, string>, structure: ExportStructure): string => {
```

Comment:
This isn't the correct code style we use. It should either be on a single line or wrapped like this if needed:
```suggestion
const legacyIndexToAdoc = (
  namespace: NavFile,
  template: HandlebarsTemplateDelegate,
  descriptions: Record<string, string>,
  structure: ExportStructure
): string => {
```

Reply:
Ah I was wondering about that, I'll fix it up.